pyproject.toml

```diff
@@ -38,6 +38,7 @@ plugins = ["django_coverage_plugin"]
 omit = [
   "*/assets/*",
   "airlock/lib/git.py",
+  "tests/functional/test_docs_screenshots.py",
```

Comment:
It always mildly annoys me that there's no comment you can stick in the header of a file to get coverage to ignore the whole thing. The nice thing about pragma comments is that they live in-line with the code rather than in some external configuration. Oh well!